T2Wil/domestic_workers_system_front_end
# #172183079 Setting up  the environment

#### What does this PR do
This PR initializes the project setup.

#### Description of the task to be completed
- confiture react
- configure redux
- configure bootstrap
- configure sass
- confiture webpack
- confiture babel
- configure eslint

#### How should this manually be tested?
- clone the git repo.
- cd into the repo
- run `npm i` to install the dependencies
- run `npm start`  to run the web-pack server

after starting the web pack server, the browser should open with the following content :
![image](https://user-images.githubusercontent.com/30593186/78624860-ec176e80-788a-11ea-8672-32f4d9508742.png)

#### Relevant Pivotal tracker stories
[link](https://www.pivotaltracker.com/story/show/172183079)
